packages/analytics-nextjs/src/events.ts

```diff
@@ -3,13 +3,21 @@ export const ACTIONS = {
     SEARCH: 'Search',
     SUBSCRIBE_FORM_SUBMIT: 'Subscribe form submit',
     SWITCH_LANGUAGE: 'Switch language',
+    COPY_STORY_LINK: 'Newsroom - Story Page - Story link copy',
```

Comment:
Is there a reason to have the `Newsroom` prefix? Seems redundant considering all these events are fired from the newsroom.

Reply:
I'm also thinking the property naming makes more sense than the actual event name. `Copy story link` sounds more human than `Story link copy`. Applies to other introduced events as well.

Reply:
Make more sense to me too. I just followed the request of @iamhugofelix: https://linear.app/prezly/issue/DEV-19287/sharing-tracking#comment-bf0f204f

@iamhugofelix are you ok if we switch to Lukas' suggestion?

Reply:
This is how it was implemented in the sign-up trackers. The reasons why this is like this are explained on this [Linear thread](https://linear.app/prezly/issue/DEV-13605/review-trackers-and-create-a-better-mixpanel-report#comment-238c55e0).

Reply:
The major difference here is that these events here are not going into Mixpanel, they are going into Plausible and our analytics API, whereas the events that we're tracking in the app are going into Mixpanel.

I don't really have a strong opinion on this so you can keep it like that, but I would at least improve the wording.

Reply:
No strong opinion either. Please proceed with what you find better.

Reply:
418f5a709239e71ed531ae6ba53c2025983535b2

Reply:
Also keep in mind that these events, if we decide to expose them in the Plausible dashboard for our users, have to be used as-is. It's not possible to change the event names for the UI (at least last time we've done this), so users will possibly see these long event names in their dashboards. Hence why I was also proposing shortening them.